### PR TITLE
Drop Phing for Makefile

### PR DESCRIPTION
**Pull request by Kocal**
See https://github.com/phpstan/phpstan-symfony/pull/160

### CODE REVIEW

File: .github/workflows/lock-closed-issues.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: 'Lock Issues'
```
Comment (ondrejmirtes) [inline .github/workflows/lock-closed-issues.yml]: Why is this file showing up here again when it's already present on master? https://github.com/phpstan/phpstan-mockery/blob/master/.github/workflows/lock-closed-issues.yml Perhaps this branch isn't rebased?

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

**Comment by ondrejmirtes** [comment]
So the work remaining is:

* phpstan-mockery
* phpstan-nette
* phpdoc-parser https://github.com/phpstan/phpdoc-parser

Additionally, if you feel like it, it would be great if all extensions got the following improvements:

* Run PHPCS on PHP 8
* Downgrade PHPUnit on older PHP versions instead of upgrading it on newer PHP versions. This is so that extensions can be developed on newer PHP versions out of the box. See how I did it in phpstan-src: https://github.com/phpstan/phpstan-src/commit/8db956fe88ced302d44ab65a4bd1168b93e02f25#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34 (the work will be easier in extensions because they don't use paratest)

File: .github/workflows/lock-closed-issues.yml (hunk 1 above)
Comment (Kocal) [reply .github/workflows/lock-closed-issues.yml]: Weird, I've forked the repository this morning. 
Rebasing.

**Comment by Kocal** [comment]
Alright, I think I would be able to work on it monday or tuesday.

**Pull request merged**